specification/paths/RegisteredShipments.json

```diff
@@ -30,6 +30,31 @@
               "meta": {
                 "type": "object",
                 "properties": {
+                  "collection": {
+                    "type": "object",
+                    "properties": {
+                      "name": {
+                        "type": "string"
+                      },
+                      "collection_date": {
+                        "type": "string",
+                        "format": "date-time"
```

Comment:
I think this should be `date` (if that format means Y-m-d) because the time is specified in `collection_time`.